src/libraries/System.Net.HttpListener/src/System.Net.HttpListener.csproj

```diff
@@ -247,7 +248,7 @@
     <Compile Include="$(CommonPath)Interop\Windows\SspiCli\SSPIWrapper.cs"
              Link="Common\Interop\Windows\SspiCli\SSPIWrapper.cs" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetsUnix)' == 'true' or '$(TargetsBrowser)' == 'true' or '$(ForceManagedImplementation)' == 'true'">
+  <ItemGroup Condition="'$(TargetsUnix)' == 'true' or '$(ForceManagedImplementation)' == 'true' and '$(TargetsBrowser)' != 'true'">
```

Comment:
I was going to suggest the same thing all of the checks below suggest: add parentheses around this. 😸

Reply:
I think we can just remove `'$(TargetsBrowser)' == 'true'` no need to add `'$(TargetsBrowser)' != 'true'`

Reply:
done